src/LocationHistory.py

```diff
@@ -175,3 +185,8 @@ def RemoveErroneousAltitude(KmlTree: ET.ElementTree) -> None:
             if Coordinates is not None and Coordinates.text is not None and re.search(RegEx, Coordinates.text):
                 # All altitudes are 0. Remove them.
                 Coordinates.text = re.sub(CoordinatesRegEx, '\\1', Coordinates.text)
+
+
+def DisplayProgress(date: DT.date, current_download: int, total_downloads: int):
+    """Display Current Progress"""
+    print(f'Downloading {date} | {current_download}/{total_downloads} | {current_download/total_downloads*100:.2f}%\r', end='', flush=True)
```

Comment:
I specifically left this off so that the progress line will be overwritten rather than having multiple showing the progress updates.  

ie, this:

```
Downloading yyyy-mm-dd | n/10 | x.xx%
```

instead of 

```
Downloading yyyy-mm-dd | 1/10 | 10.00%
Downloading yyyy-mm-dd | 2/10 | 20.00%
Downloading yyyy-mm-dd | 3/10 | 30.00%
Downloading yyyy-mm-dd | 4/10 | 40.00%
etc.
```

Do you still want me to add the newline character?

Reply:
I actually meant at the end of the file, not the end of the print :)

Reply:
Ah, that makes a lot more sense!  Sorry about that.